src/utils/ethereum.js

```diff
@@ -87,22 +88,37 @@ export const getCoinDetails = async (
   scDecimals,
   rcDecimals
 ) => {
+  let promiseArray;
+
+  if (NETWORK_ID === "200101" || NETWORK_ID === "2001") {
```

Comment:
This solutions is strange, actually. We shouldn't have to have an "if" statement for every blockchain here. The code should be the same for all chains and we should just have to change the parameters for each chain in the ".env" file.

It seems to me that either `BC_DECIMALS` or `scDecimals` is configured incorrectly for Milkomeda or Sepolia and, instead of fixing this in the env file, we are hacking it here, in the wrong place.

@Tanya-ruby , I will merge this PR, because we need to launch djed on sepolia. But could you try to submit another PR with a better solution?

Reply:
@ceilican 
I've thoroughly logged all the values ,the conclusion is that the values fetched from sepolia are significantly smaller than milkomeda.Eg: 300 in sepolia and on milkomeda it will be 30000000 .
Also the BC_DECIMALS,scDecimals,rcDecimals remain the same irrespective of the network .It depends on the functions that take them as the argument .
1)Not all functions in sepolia take the scDecimals ,they take BC_Decimals as well .
Eg:To display the total supply- milkomeda takes into consideration the scDecimals but sepolia takes in the BC_DECIMALS.
In another case,sepolia takes in sc_Decimals to display the value of price of stablecoin instead of BC_Decimals like the milkomeda.
We cannot configure the env file based on it ,We will have to manually check the value fetched and then customise the function to either take sc/bc decimals.
sc/bc decimals are same for all the networks so modifying them in the env file will make no difference.We just have to modify the parameters taken into consideration by the function.
I've attached a screenshot on discord explaining the same !